lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -521,14 +522,20 @@ def build_image(self):
         for line in self.client.build(**params):
             # line = json.loads(line)
             self.log(line, pretty_print=True)
+            if "stream" in line:
+                build_output.append(line["stream"])
             if line.get('error'):
                 if line.get('errorDetail'):
                     errorDetail = line.get('errorDetail')
-                    self.fail("Error building %s - code: %s message: %s" % (self.name,
-                                                                            errorDetail.get('code'),
-                                                                            errorDetail.get('message')))
+                    self.fail(
+                            "Error building %s - code: %s, message: %s, logs: %s" % (
+                            self.name,
```

Comment:
Please fix the PEP8 errors here https://app.shippable.com/runs/589091484862ac1000f7f0a0/1/console